adapter/pkg/messaging/azure_connection.go

```diff
@@ -29,7 +29,7 @@ import (
 	logger "github.com/wso2/product-microgateway/adapter/internal/loggers"
 )
 
-var bindingKeys = []string{tokenRevocation, notification, stepQuotaThreshold, stepQuotaReset}
+var bindingKeys = []string{tokenRevocation, notification, stepQuotaThreshold, stepQuotaReset, organizationPurge}
```

Comment:
@Rajith90 AFAIR you had a concern with this array based approach for connection keys?

Reply:
Yes, we decided to refactor this when @erandi refactors the rate limiting related topics

Reply:
@erandiganepola could you please follow up on this